vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectLinksComponent.java

```diff
@@ -103,7 +120,7 @@ public List<String> linkedOffers() {
   }
 
   public void setStyles(String... componentStyles) {
-    getContent().addClassNames(componentStyles);
+    addClassNames(componentStyles);
   }
```

Comment:
what does this method do?

Reply:
It adds css styles or Lumo stuff from the outside. It's not used at the moment - should I remove it?

Reply:
Ha, okay, I see. Since we are not using this Component with a wrapper anymore, this is not needed.